Documentation/helm-operator.md

```diff
@@ -107,8 +107,10 @@ The following tables lists the configurable parameters of the rook-operator char
 | `agent.mountSecurityMode`    | Mount Security Mode for the agent.                                                                      | `Any`                                                  |
 | `agent.toleration`           | Toleration for the agent pods                                                                           | <none>                                                 |
 | `agent.tolerationKey`        | The specific key of the taint to tolerate                                                               | <none>                                                 |
+| `agent.nodeAffinity`         | The node labels for affinity of `rook-agent`                                                            | <none>                                                 |
```

Comment:
Could you add some additional docs for how to use these options either inline or in a separate section in this document.

Reply:
added.